app/views/my_modules/modals/_update_repository_records_modal_content.html.erb

```diff
@@ -0,0 +1,56 @@
+<%
+  rows_to_assign = selected_rows.select{|k, v| v[:assigned] == "true"}
+  rows_to_unassign = selected_rows.select{|k, v| v[:assigned] == "false"}
+%>
+<div class="modal-header">
+  <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+    <span aria-hidden="true">×</span>
+  </button>
+  <h4 class="modal-title">
+    <%=t 'my_modules.modals.update_repository_record.title',
+         repository_name: repository.name,
+         my_module_name: my_module.name %>
+  </h4>
+</div>
+<div class="modal-body">
+  <p><%=t 'my_modules.modals.update_repository_record.message', size: selected_rows.keys.count %></p>
```

Comment:
space

Reply:
can there be a key without a value?

Reply:
no

Reply:
so why are you counting keys?

Reply:
it is Parameters object, which doesn't have count or length methods. So i need read keys or convert to hash.

Reply:
Are you sure?

Reply:
You assume `selected_rows.values.count` ?